tests/ci/aml_cpu_unit_tests.yml

```diff
@@ -0,0 +1,73 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+#
+variables:
+  - group: AML-config
+  - name: "test"
+    value: "tests/ci/run_pytest.py"
+  - name: "testfolder"
+    value: "./tests/unit"
+  - name: "testmarkers"
+    value: '"not notebooks and not gpu and not azureml"'
+  - name: "junitxml"
+    value: "reports/test-unit.xml"
+  - name: "maxnodes"
+    value: 4
+  - name: "reponame"
+    value: "nlp"
+  - name: "clustername"
```

Comment:
We could review these parameters, maybe they are creating noise. If the parameter is used several times, then it makes sense to have a variable. However, if the parameter is used once, we will have:
```
  - name: "junitxml"
    value: "reports/test-unit.xml"

python tests/ci/submit_azureml_pytest.py [...] --junitxml $(junitxml)
```
That for every parameter. However, if we remove the non-repeated variables there will be only one line:
```
python tests/ci/submit_azureml_pytest.py [...] --junitxml "reports/test-unit.xml
```
I think the second option helps with readability. 

I'm looking at [other pipelines](https://github.com/microsoft/LightGBM/blob/master/.vsts-ci.yml) and I don't see that many options.

Do people prefer to have everything parametrized? @yueguoguo @saidbleik @jiata